google-cloud-contrib/google-cloud-logging-adapters/LICENSE

```diff
@@ -0,0 +1,201 @@
+                                 Apache License
```

Comment:
I don't think you need a license file in this subdirectory, because the root of google-cloud-java has one.